man/fifelse.Rd

```diff
@@ -15,12 +15,16 @@
 }
 \details{
 In contrast to \code{\link[base]{ifelse}} attributes are copied from the first non-\code{NA} argument to the output. This is useful when returning \code{Date}, \code{factor} or other classes.
+
+Unlike \code{\link[base]{ifelse}}, \code{fifelse} evaluates both \code{yes} and \code{no} arguments for type checking regardless of the result of \code{test}. This means that neither \code{yes} nor \code{no} should be recursive function calls. For recursiveness, use \code{fcase} instead.
```

Comment:
recursiveness -> recursion

Reply:
yes please add an example of recursion with fcase, to clarify what you mean here